core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -24,6 +24,7 @@ var ruleTemplates = require(
   '../../../extensions/interactions/rule_templates.json');
 var waitFor = require('../protractor_utils/waitFor.js');
 var action = require('./action.js');
+const { element } = require('protractor');
```

Comment:
I don't think this should be needed. The tests used `element` just fine without this import, right?

Reply:
Removed